tests/python-pytest/onnx/backend.py

```diff
@@ -123,7 +124,10 @@ def run_node(cls, node, inputs, device='CPU'):
         mod.bind(for_training=False, data_shapes=data_shapes, label_shapes=None)
 
         # initializing parameters for calculating result of each individual node
-        mod.init_params()
+        if arg_params is None or aux_params is None:
```

Comment:
If arg_params is not None and aux_params are None?

Reply:
Will fix this logic